src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -301,3 +301,27 @@ export const onRequest = defineMiddleware(async (context, next) => {
 </p>
 
 `deserializeActionResult()` will reverse the effect of `serializeActionResult()` and return an action result to its original state. This is useful to access the `data` and `error` objects on a serialized action result.
+
+### `getActionPath()`
+
+<p>
+
+**Type:** `(action: ActionClient<any, any, any>) => string`
+<Since v="5.1.0" />
+</p>
+
+The `getActionPath()` utility is used to get the path to an action, taking base and trailing slash settings into account. This is useful when you want to call an action for more advanced usages, such as providing custom headers or not using the fetch API (eg. `navigator.sendBeacon`):
+
+```ts
```

Comment:
Can we have a "this example shows..." and then describe what is happening here?

Reply:
Is it worth showing 2 examples? Eg. one using the fetch API and one using the `navigator.sendBeacon` API